cmd/cloud-network-config-controller/main.go

```diff
@@ -91,12 +98,31 @@ func main() {
 		RetryPeriod:     26 * time.Second,
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStartedLeading: func(ctx context.Context) {
+				featureGateAccessor, err := createFeatureGateAccessor(
+					ctx,
+					cfg,
+					"cloud-network-config-controller",
```

Comment:
cloud-network-config-controller has `CONTROLLER_NAME` and `CONTROLLER_NAMESPACE` envs available that we could use here instead of the hard coded strings.
https://github.com/openshift/cluster-network-operator/blob/370f8f3b186ffd9f3f78ab9f79839f70a25d84fb/bindata/cloud-network-config-controller/self-hosted/controller.yaml#L51-L58